python/fastexcel/__init__.py

```diff
@@ -61,22 +61,39 @@ def sheet_names(self) -> list[str]:
         """The list of sheet names"""
         return self._reader.sheet_names
 
-    def load_sheet_by_name(self, name: str) -> ExcelSheet:
+    def load_sheet_by_name(
+        self,
+        name: str,
+        header_row: None | int = 0,
+        column_names: None | List[str] = None,
```

Comment:
```suggestion
        header_row: int | None = 0,
        column_names: list[str] | None = None,
```

Convention + native types